Sources/NiceComponents/Helper/Text+AttributedString.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Text+AttributedString.swift
+//  
+//
+//  Created by Alejandro Zielinsky on 2022-06-08.
+//
+
+import SwiftUI
+
+extension Text {
```

Comment:
I'm a little hesitant to add built in support for NSAttributed strings in here for a couple reasons:
- It goes against the SwiftUI paradigm of being declarative
- It seems like you may just be able to [use Markdown](https://stackoverflow.com/a/67922456) now, though we may need to [adjust our constructors a little](https://developer.apple.com/forums/thread/683632)
- I'm not thrilled with the manual unwrapping/application we're doing for attributes, while we are covering the most common ones I think it's potentially confusing/frustrating to support some but not all


At the very least, I think it's worth removing this from this PR and evaluating it on its own.

Reply:
I agree that it kinda sucks to still be using NSAttributedString but this seems to be the best/most convenient option for supporting sub iOS 15 versions. If we want to bump up the min iOS version for the NiceComponents library to be iOS 15 then I found a much better option using AttributedString. [Link](https://stackoverflow.com/a/67889368 )

Reply:
I think that's a big enough call to warrant a separate ticket/PR

Reply:
Ok i'll take it out of this PR so we can discuss further. Although, I personally think it's a useful extension to include as Apple is unlikely to deprecate NSAttributedString anytime soon and any other option requires us to support iOS 15+. 
We're also already using this extension on Ridwell and GpsGuide.